addon/services/crowdin.js

```diff
@@ -0,0 +1,32 @@
+import Service from '@ember/service';
+import injectScript from 'ember-cli-crowdin/utils/inject-script';
+import { getOwner } from "@ember/application";
+
+export default Service.extend({
+  isInContextLoaded: false,
+
+  init() {
+    this._super(...arguments);
+    this.config = getOwner(this).resolveRegistration('config:environment');
+  },
+
+  enableInContext() {
+    this._setup();
+    this._enable();
+    this.set('isInContextLoaded', true);
```

Comment:
this needs to be in the ` this._enable().then(()=> { this.set('isInContextLoaded', true)});`

Reply:
I had to move the `then` to inside the `_enable` method cause thats were the promise resolves.